src/content/blog/whats-new-february-2024.mdx

```diff
@@ -0,0 +1,195 @@
+- [inline-mod-astro](https://inox-tools.vercel.app/inline-mod) by @fryuni
+- [JulianCataldo/og-images-generator](https://github.com/JulianCataldo/og-images-generator#astro-integration) by @juliancataldo
+- [BryceRussell/astro-pages](https://github.com/BryceRussell/astro-pages) by @bryceguy
+- [ktym4a/astro-page-insight](https://github.com/ktym4a/astro-page-insight) by @ktym4a
+- [kindspells/astro-sri-scp](https://www.npmjs.com/package/@kindspells/astro-sri-csp) by @andreu
+- [Tiny toolbar app to show Tailwind Breakpoints](https://elk.zone/m.webtoo.ls/@zero_gravitas@indieweb.social/111925333428424557)	
+- [frankstallone/astro-cube-boilerplate](https://github.com/frankstallone/astro-cube-boilerplate) by @frank_stallone
+- [KindSpells/astro-shield](https://github.com/KindSpells/astro-shield) by @castarco
+- [natemoo-re/astro-remote](https://github.com/natemoo-re/astro-remote)	by @nmoodev
+- [Generate e-mails with Astro](https://github.com/luckydye/astro-email/tree/main) by @luckydye
+
+### Themes & Templates
+
+Several new Astro themes were added & released this month. Try them out!
+
+- [Illustration based template](https://github.com/manulthanura/Illustration-Based-Portfolio) by @manulthanura
+- [Astroplate](https://astro.build/themes/details/astroplate/) by @tffarhad
+- [Quantum](https://cosmicthemes.com/themes/quantum/) by @lastchaos.
+- [event-edge.gerome.dev](https://event-edge.gerome.dev/) by @gerome.dev
+- [ScrewFast](https://github.com/mearashadowfax/ScrewFast) by @soslan
+- [MinimalStudio, theme built with Astro, TailwindCSS and Alpine](https://minimalstudio.vercel.app/) by @mike_andreuzza
+- [Resume website with TinaCMS](https://astro-resume02.vercel.app/) by @alpeshnakrani
+
+### Websites
+
+All the community member websites submitted to our Discord `#showcase` channel and featured on community calls this month.
+
+export const sites = [
+  { url: 'https://easy-csv.netlify.app/profile', title: '@christopheanfry'},
+  { url: 'https://evadecker.com/hypersensitive', title: '@for.eva.new'},
+  { url: 'https://warawul.coffee/en/', title: '@chris_origins_coffee'},
+  { url: 'https://lindakat-arcade.netlify.app/', title: '@lindakat'},
+  { url: 'https://daniell.dev/', title: '@daniellwdb'},
+  { url: 'https://heschl.dev/', title: '@joneswastaken'},
+  { url: 'https://www.benjamindorey.fr/', title: '@biommxx'},
+  { url: 'https://pro-g.eu/', title: '@biommxx'},
+  { url: 'https://hitesh.is-a.dev/', title: '@hitesh_dev'},
+  { url: 'https://www.tekac.club/humans', title: '@emptybrother7'},
+  { url: 'https://evadecker.com/design', title: '@for.eva.new'},
+  { url: 'https://joelmturner.com/', title: '@joelmturner'},
+  { url: 'https://hermes.cadens.studio/', title: '@cadens.studio'},
+  { url: 'https://barvian.me/', title: '@mbarvian'},
+  { url: 'https://www.deskpro.com/', title: '@mattheous'},
+  { url: 'https://wybm-valentine.netlify.app/', title: '@withered.flowers'},
+  { url: 'https://mike-astrotutorial.netlify.app/', title: '@mikeoxygen'},
```

Comment:
This one seems to be the Astro tutorial, not sure if expected to be showcased?

Reply:
Yeah, all Mike did was change his name on the about page... so we can probably skip this one!